.github/pull_request_template.md

```diff
@@ -13,34 +14,35 @@ It may include
 
 The summary may omit some of these details if this PR fixes a single ticket that includes these details. It is the reviewer's discretion. 
 
-## [Optional] Changes
-Multiple changes are not recommended, so this section should normally be omitted. Unforunately, they are sometimes unavoidable. If there are multiple logical changes, list them separately.
+## Changes [Optional] 
+Multiple changes are not recommended, so this section should normally be omitted. Unfortunately, they are sometimes unavoidable. If there are multiple logical changes, list them separately.
 
 1. `'foo'` is replaced by `'bar'`
 2. `'fizzbuzz'` is optimized for gas
 
-# [Optional] How Has This Been Tested?
+## How Has This Been Tested? [Optional] 
 
 Did you need to run manual tests to verify this change? If so, please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration
 
-# [Optional] :warn: Does this require multiple approvals?
+## :warning: Does this require multiple approvals? [Optional]
 Please explain which reason, if any, why this requires more than one approval.
 - [ ] Is it security related?
 - [ ] Is it a significant process change?
 - [ ] Is it a significant change to architectural, design?
 
-# Checklist:
+---
+## Checklist:
 
-## Code quality
+### Code quality
 - [ ] I have written clear commit messages
 - [ ] I have performed a self-review of my own code
-- [ ] I have scoped this change as narrowly as possible
```

Comment:
Taking this PR as a (slightly contrived) example: I could have put each atomic change in its own PR: but that seems like overkill. I think the author can mostly just use their judgement to manage the scope of a PR.

I also think that, rather than trying to pre-empt the desires of the reviewer, we can allow the reviewer to ask the originator to split a PR if they deem the scope to be too wide. This point goes for some of the other prompts in the template, too: a reviewer can and should ask for more context if they need it, for example.